pkg/minikube/cluster/cluster.go

```diff
@@ -248,12 +255,29 @@ func (m *MachineConfig) CacheMinikubeISOFromURL() error {
 		}()
 	}
 
+	if _, err = io.Copy(tmpISOFile, iso); err != nil {
```

Comment:
For what it's worth, using something like ISO filename + ".part" might be a bit more effective. You basically download already into the right directory, just with the suffix '.part' or '.tmp'. Once the download is complete and verified you just rename the file. Renaming is cheaper than another copy and probably less error prone. I guess we can do either one.